uint32 robot_id
---
dpgo_ros/MatrixMsg matrix